src/dataAPI/jupyter/PythonExecutor.ts

```diff
@@ -206,10 +206,11 @@ export class PythonPandasExecutor {
      */
     private async getDFVars(varNames: string[]): Promise<string[]> {
         try {
-            const code_lines = ['import pandas as pd']; // TODO better way to make sure pandas in env?
+            const code_lines = ['import pandas as __autoprofilerPandas']; // TODO better way to make sure pandas in env?
             varNames.forEach(name =>
-                code_lines.push(`print(type(${name}) == pd.DataFrame)`)
+                code_lines.push(`print(type(${name}) == __autoprofilerPandas.DataFrame)`)
             );
+            code_lines.push('del __autoprofilerPandas')
```

Comment:
This del is probably unnecessary since if user defined a variable as `__autoprofilerPandas` it will have already been overwritten